### PR TITLE
Kotlin tan

### LINKED ISSUE #3632: [Term Entry] Kotlin Math .tan()
### Reason/inspiration (optional)

We would like a new term entry on the `.tan()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/tan/tan.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by Anudhyan**
### Description

- Created a new term entry for tan() in Kotlin Programming language

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

Fixes #3632

### CODE REVIEW

**Comment by Anudhyan** [comment]
@SSwiniarski sir it is failing one check

File: content/kotlin/concepts/math-methods/terms/tan/tan.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+tan(x)
+```
+
+The function accepts one mandatory parameter `x`. They all take an angle in radians as `Double` or `Float` as a parameter.
+
+The function returns a `Double` value, which is the tangent of the angle.
+
+## Example
+
+The example below demonstrates how to use the tan() function in Kotlin calculates the tangent of the angle `45` degrees in radians.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/tan/tan.md:31]:
```suggestion
The example below demonstrates how to use the `tan()` function in Kotlin to calculate the tangent of the angle `45` degrees in radians.

```

File: content/kotlin/concepts/math-methods/terms/tan/tan.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+```kotlin
+import kotlin.math.*
+fun main() {
+    val angleInRadians = Math.toRadians(45.0)
+    val tangent = tan(angleInRadians)
+
+    println(tangent) // 1.0
+}
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/tan/tan.md:40]:
```suggestion
import kotlin.math.*
fun main() {
  val angleInRadians = Math.toRadians(45.0)
  val tangent = tan(angleInRadians)

  println(tangent)
}
```

File: content/kotlin/concepts/math-methods/terms/tan/tan.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## Syntax
+
+```pseudo
+tan(x)
+```
+
+The function accepts one mandatory parameter `x`. They all take an angle in radians as `Double` or `Float` as a parameter.
+
+The function returns a `Double` value, which is the tangent of the angle.
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/math-methods/terms/tan/tan.md:27]:
```suggestion
- The function accepts one mandatory parameter `x`. They all take an angle in radians as `Double` or `Float` as a parameter.
- The function returns a `Double` value, which is the tangent of the angle.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side

**Comment by dakshdeepHERE** [comment]
@Anudhyan Don't worry about the checks

**Comment by Anudhyan** [comment]
> @Anudhyan Don't worry about the checks

Hello @dakshdeepHERE I have applied the changes . Please check now and accept the pull request by adding the hacktoberfest tag as it was originally there

**Comment by dakshdeepHERE** [comment]
@Anudhyan Done, This LGTM for 2nd review. 🚀

**Review by dakshdeepHERE — APPROVED** [review]

**Comment by Anudhyan** [comment]
> Reference i
@dakshdeepHERE please tag Someone who can review it asap and merge the pull request

**Comment by dakshdeepHERE** [comment]
@Anudhyan Please have patience our other maintainers live in different time zones. So next reviews can take time. Don't worry.

**Comment by Anudhyan** [comment]
> @Anudhyan Please have patience our other maintainers live in different time zones. So next reviews can take time. Don't worry.

I do understand this but hacktoberfest ends today so I am a bit nervous & anxious

**Comment by Anudhyan** [comment]
@caupolicandiaz @SSwiniarski sir kindly check this

**Review by SSwiniarski — APPROVED** [review]
@Anudhyan Looks good.

**Comment by Anudhyan** [comment]
> @Anudhyan Looks good.

Thanks sir for the motivation . I feel so confident now !

**Pull request merged**